temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -217,6 +217,40 @@ message DeleteNamespaceResponse {
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
 
+message FailoverNamespaceRegionRequest {
+    // The namespace to failover.
+    string namespace = 1;
+    // The id of the region to failover to.
+    // Must be a region that the namespace is currently available in.
+    string region = 2;
+    // Skip graceful failover and only start forceful failover.
+    bool skip_graceful_failover = 3;
+    // The id to use for this async operation - optional.
+    string async_operation_id = 4;
+}
+
+message FailoverNamespaceRegionResponse {
+    // The async operation.
+    temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
+}
+
+message AddNamespaceRegionRequest {
+    // The namespace to add region to.
+    string namespace = 1;
+    // The id of the region to add to the namespace.
```

Comment:
Do we need any clarification comment here that this is secondary/passive region?